argocon-demos/2026-dont-do-what-donny-dont-does-the-10-dos-and-500-donts-of-workflows-success/README.md

```
@@ -0,0 +1,31 @@
+# Don't Do What Donny Don't Does: The 10 Dos and 500 Don'ts of Workflows Success
+
+[![Pipekit Logo](https://raw.githubusercontent.com/pipekit/talk-demos/main/assets/images/pipekit-logo.png)](https://pipekit.io?utm_campaign=talk-demos)
+
+
+<!-- ## The talk
+The talk recording [can be found here](https://youtu.be/0QeYA6_9cOg).-->
+
+The slide deck for this talk can be found [here](assets/slide-deck.pdf).
```

markdownlint-cli2 (0.21.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)